src/User/pages/Privacy-Policy/Privacy.jsx

```
@@ -328,56 +303,65 @@ const Privacy = () => {
+            </h2>
+            <p style={{ marginTop: "4px", marginBottom: "0px" }}>
               If you have a question, issue, complaint, or feedback in relation
               to the processing of your personal data or this Privacy Policy,
               you may reach out to our data protection officer at{" "}
               <a
                 className="a"
                 href="mailto:grievanceofficer@vigybag.com"
-                target="_blank">
-                grievanceofficer@vigybag.com.
+                target="_blank"
+              >
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)